src/components/src/B2cHomePage/b2c.home.page.tsx

```diff
@@ -55,87 +56,109 @@ const B2CHomePage: React.FunctionComponent = () => {
+              </div>
+            </li>
+            <li className="main-goods__cell">
+              <div className="main-goods-wrap">
+                <div className="goods-info">
+                  <h5 className="goods-title-small">{intl.get('b2c-product3-label')}</h5>
+                  <h3 className="goods-title">{intl.get('b2c-product3-heading')}</h3>
+                  <p className="goods-description">
+                    {intl.get('b2c-product3-description')}
+                  </p>
+                  <button type="button" className="ep-btn primary learn-more-btn">{intl.get('b2c-product3-label')}</button>
+                </div>
+                <ImageContainer className="main-goods-image" fileName={iceCreamImageFileName} imgUrl={iceCreamImage} />
+              </div>
+            </li>
+            <li className="main-goods__cell">
+              <div className="main-goods-wrap">
+                <div className="goods-info">
+                  <h5 className="goods-title-small">{intl.get('b2c-product4-label')}</h5>
+                  <h3 className="goods-title">{intl.get('b2c-product4-heading')}</h3>
+                  <p className="goods-description">
+                    {intl.get('b2c-product4-description')}
+                  </p>
+                </div>
+                <ImageContainer className="main-goods-image" fileName={shippingImageFileName} imgUrl={heroImageShipping} />
+              </div>
+            </li>
+            <li className="main-goods__cell">
+              <div className="main-goods-wrap">
+                <div className="goods-info">
+                  <h5 className="goods-title-small">{intl.get('b2c-product5-label')}</h5>
+                  <h3 className="goods-title">
+                    {intl.get('b2c-product5-heading')}
+                  </h3>
+                </div>
+                <ImageContainer className="main-goods-image" fileName={ARImageFileName} imgUrl={ARImage} />
+              </div>
+            </li>
+          </ul>
         </div>
-        <div className="section-parallax section-parallax-2 container" data-region="homeMainContentRegion">
-          <Parallax y={[10, -30]} tagOuter="figure">
-            <div className="parallax-image-container">
-              <ImageContainer className="parallax-image" fileName={homeEspotParallax2FileName} imgUrl={homeEspotParallax2} />
-            </div>
-          </Parallax>
-          <div className="sub-section">
-            <div className="text-block">
-              <span className="line line-white">
-                {intl.get('home-sub-espot-container2-second-line')}
-              </span>
-              <span className="line line-link">
-                {intl.get('learn-more')}
-              </span>
+      </div>
+
+      <section className="main-banner">
+        <ImageContainer className="main-banner-image" fileName={heroImageToasterFileName} imgUrl={heroImageToasterImage} />
```

Comment:
This image is fairly low res. Let's check with Bonnie if any others exist or we can do something about the current.